keyword-guidelines.md

```diff
@@ -0,0 +1,102 @@
+# Keyword Guidelines for the Specifications
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" as used in the specification
+documents are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997.)
+      
+## Introduction
```

Comment:
I think adding the reference (and link) to RFC2119 is good enough. I don't think we should repeat it here, so I'd remove the text below.  Repeating it means that if, heaven forbid, there's a typo in there we'd have to explain which one wins and that's just annoying.  Avoid the entire issue by just having a reference.